source/dub/recipe/packagerecipe.d

```diff
@@ -250,9 +250,17 @@ struct BuildSettingsTemplate {
 						continue;
 					}
 
-					foreach (d; dirEntries(path.toNativeString(), pattern, SpanMode.depth)) {
-						import std.path : baseName;
-						if (baseName(d.name)[0] == '.' || d.isDir) continue;
+					auto pstr = path.toNativeString();
+					foreach (d; dirEntries(pstr, pattern, SpanMode.depth)) {
+						import std.path : baseName, pathSplitter;
+						import std.algorithm.searching: canFind;
+						import std.string: startsWith;
```

Comment:
unused import

Reply:
Yep. I think I intended to use it and then just used `name[0] == '.'`